### PR TITLE
feat(data-table): slug

### LINKED ISSUE #11139: [CWC Data Table] Add AI enhancements
> [!WARNING]  
> **Blocked by**
>
> - https://github.com/carbon-design-system/carbon/issues/15189

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

Closes #11139 

### Description

slug options to data table 
### Changelog

**New**

- data table story folder in slug

**Changed**

- {{changed thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: packages/carbon-web-components/src/components/slug/slug-data-table-story.ts [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
+        <cds-table-expanded-row>
+          <h6>Expandable row content</h6>
+          <div>Description here</div>
+        </cds-table-expanded-row>
+      </cds-table-body>
+    </cds-table>
+  `;
+};
+
+export const _SlugWithRadio = () => {
```
Comment (annawen1) [inline packages/carbon-web-components/src/components/slug/slug-data-table-story.ts:338]:
```suggestion
export const _SlugWithRadioSelection = () => {
```

just to match with the React storybook

**Review by aagonzales — CHANGES_REQUESTED** [review]
1. Header label should have 16px padding on the left side.

As-is: 
image


To-be:
image


2. Radio and checkbox selection is missing gradient on selected 

As-is: 
image

To-be:
image

**Review by annawen1 — APPROVED** [review]
Code-wise, LGTM!

**Comment by ariellalgilmore** [comment]
@aagonzales thanks for the review! the deploy preview should be updated with the correct changes now

**Review by aagonzales — APPROVED** [review]
Looks good to go now! 🎉 ✅

**Pull request merged**